hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/AtomicCryptoServiceFeesSuite.java

```diff
@@ -0,0 +1,550 @@
+                        .has(accountDetailsWith()
+                                .cryptoAllowancesCount(2)
+                                .nftApprovedForAllAllowancesCount(1)
+                                .tokenAllowancesCount(2)
+                                .cryptoAllowancesContaining(SECOND_SPENDER, 100L)
+                                .tokenAllowancesContaining(FUNGIBLE_TOKEN, SECOND_SPENDER, 100L)),
+                /* edit existing allowances */
+                atomicBatch(cryptoApproveAllowance()
+                                .payingWith(OWNER)
+                                .addCryptoAllowance(OWNER, SECOND_SPENDER, 200L)
+                                .via("approveModifyCryptoTxn")
+                                .fee(ONE_HBAR)
+                                .blankMemo()
+                                .logged()
+                                .batchKey(batchOperator))
+                        .via("atomicBatch")
+                        .signedByPayerAnd(batchOperator),
+                validateInnerTxnChargedUsd("approveModifyCryptoTxn", "atomicBatch", 0.049375, 5),
+                atomicBatch(cryptoApproveAllowance()
+                                .payingWith(OWNER)
+                                .addTokenAllowance(OWNER, FUNGIBLE_TOKEN, SECOND_SPENDER, 200L)
+                                .via("approveModifyTokenTxn")
+                                .fee(ONE_HBAR)
+                                .blankMemo()
+                                .logged()
+                                .batchKey(batchOperator))
+                        .via("atomicBatch")
+                        .signedByPayerAnd(batchOperator),
+                validateInnerTxnChargedUsd("approveModifyTokenTxn", "atomicBatch", 0.04943, 5),
+                atomicBatch(cryptoApproveAllowance()
+                                .payingWith(OWNER)
+                                .addNftAllowance(OWNER, NON_FUNGIBLE_TOKEN, ANOTHER_SPENDER, false, List.of())
+                                .via("approveModifyNftTxn")
+                                .fee(ONE_HBAR)
+                                .blankMemo()
+                                .logged()
+                                .batchKey(batchOperator))
+                        .via("atomicBatch")
+                        .signedByPayerAnd(batchOperator),
+                validateInnerTxnChargedUsd("approveModifyNftTxn", "atomicBatch", 0.049375, 5),
+                getAccountDetails(OWNER)
+                        .payingWith(GENESIS)
+                        .has(accountDetailsWith()
+                                .cryptoAllowancesCount(2)
+                                .nftApprovedForAllAllowancesCount(0)
+                                .tokenAllowancesCount(2)
+                                .cryptoAllowancesContaining(SECOND_SPENDER, 200L)
+                                .tokenAllowancesContaining(FUNGIBLE_TOKEN, SECOND_SPENDER, 200L)));
```

Comment:
We want to avoid really long tests. 

From the `@HapiTest` checklist in test-client's `README.md` [here](https://github.com/hiero-ledger/hiero-consensus-node/blob/e1f8b95c4af275621b4936cce9e6946a788c2250/hedera-node/test-clients/README.md?plain=1#L255):
```
The test includes no more than seven (7) `SpecOperation`'s.
```

Let's find a reasonable way to condense this test (and other tests in this class).